src/main/java/core/basesyntax/strategy/impl/SupplyOperationHandler.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.dao.FruitStorageDao;
+import core.basesyntax.dao.FruitStorageDaoImpl;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.OperationHandler;
+
+public class SupplyOperationHandler implements OperationHandler {
+    private final FruitStorageDao fruitStorageDao = new FruitStorageDaoImpl();
+
```

Comment:
```suggestion
public class SupplyOperationHandler implements OperationHandler {
    private final FruitStorageDao fruitStorageDao;

    public SupplyOperationHandler(FruitStorageDao fruitStorageDao) {
        this.fruitStorageDao = new FruitStorageDaoImpl();
    }
```
change in each handler accordingly